src/utils/messageEmbed.ts

```diff
@@ -88,19 +88,19 @@ export async function embedMessage(
                     const whileEmbed = new EmbedBuilder()
                     .setTitle(`Responding to ${message.author.tag}`)
                     .setDescription(result.slice(0, 5000) || 'No Content to Provide...')
-                    .setColor('#00FF00')
+                    .setColor('#00FF00');
```

Comment:
Don't need a semicolon here.

Reply:
In this case, a semicolon does seem to be needed in order to allow the channel to be casted as a `SendableChannels` type.

Reply:
I can try and find a better way to cast the channel as the `SendableChannels` type... maybe that means climbing up the inheritance tree a few rungs.

Reply:
Gonna investigate as well.

Reply:
Found a solution. Going to implement and push when done.